source/docs/a2.installation.md

```diff
@@ -4,9 +4,9 @@ title: Installation
 ## Rustup
 There are many ways to install Rust on your system. For the moment the official way to install Rust is using [Rustup](https://rustup.rs/).
 
-[📖](https://github.com/rust-lang-nursery/rustup.rs) Rustup installs The Rust Programming Language from the official release channels, enabling you to easily switch between **stable, beta, and nightly** compilers and keep them updated. It makes **cross-compiling** simpler with binary builds of the standard library for common platforms.
+[📖](https://github.com/rust-lang-nursery/rustup.rs) Rustup installs Rust from the official release channels, enabling you to easily switch between **stable, beta, and nightly** compilers and keep them updated. It makes **cross-compiling** simpler with binary builds of the standard library for common platforms.
```

Comment:
These were copied from official docs. I think better keep `The Rust Programming Language`